packages/dev/core/src/DeviceInput/InputDevices/deviceTypes.ts

```diff
@@ -6,7 +6,7 @@ import type { DeviceType, PointerInput, DualShockInput, XboxInput, SwitchInput,
 export type DeviceInput<T extends DeviceType> = T extends DeviceType.Keyboard | DeviceType.Generic
     ? number
     : T extends DeviceType.Mouse | DeviceType.Touch
-    ? Exclude<PointerInput, PointerInput.Move>
+    ? Exclude<PointerInput, PointerInput.Move | PointerInput.MouseWheelX | PointerInput.MouseWheelY | PointerInput.MouseWheelZ>

```

Comment:
Probably should add this to what's new under breaking changes.

Reply:
Updated